### DISCUSSION TITLE
archive/2a246caf-fb12-45cd-bad3-877174fcf2c0

### DISCUSSION THREAD

**Discussion by giscus** [macjjuni/devlog_latest#17] [Announcements]
# archive/2a246caf-fb12-45cd-bad3-877174fcf2c0

프로젝트를 시작하기 전 Hello World!를 찍어보는 것 처럼 블로그에서도 첫 글을 작성해본다. 

http://localhost:3000/archive/2a246caf-fb12-45cd-bad3-877174fcf2c0

**Comment by kkusaeng** [comment]
🏃🏻‍♂️
